src/Tools/dotnet-trace/CommandLine/Commands/Reports/TopNReport.cs

```diff
@@ -0,0 +1,135 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Tools.Common;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Binding;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.Tracing.Etlx;
+using Microsoft.Diagnostics.Symbols;
+using Microsoft.Diagnostics.Tracing.Stacks;
+using Microsoft.Diagnostics.Tracing;
+using Diagnostics.Tracing.StackSources;
+using Microsoft.Diagnostics.Tools.Trace.CommandLine;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal static class TopNReportHandler
+    {
+        private static readonly List<string> UnwantedMethodNames = new() { "ROOT", "Process"};
```

Comment:
nit: Shouldn't this be a Set instead instead of using List and using LINQ to query it ?